app/models/gem_content.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class GemContent
```

Comment:
In rubygems.org related codebase package is called `rubygem` and the naming is build around that. There is `Rubygem` model and `rubygem` relations. On the other side, this naming (most likely to represent `Gem::Content` seems using `rubygems` itself namespace `Gem`.

To keep it consistent, I propose to do one of following:

- keep it in `Gem` namespace (thinking about this as an extension to rubygems itself) and move it into `lib`. It is ok to scope it into `Gem::Content`.
- move it into `Rubygem` namespace and keep it into models

Reply:
This belongs to Rubygems.org. It's essentially a specialized RubygemFs, so it seems reasonable to change to RubygemContent.

Reply:
Changed, and I was inspired to cleanup the [plural/singular "contents" inconsistency](https://proofed.com/writing-tips/word-choice-content-vs-contents/). I think it reads better now.